yaml/thz504.yaml

```diff
@@ -1,3 +1,12 @@
+defaults:
+  clk_pin: "GPIO14"
+  mosi_pin: "GPIO13"
+  miso_pin: "GPIO4"
+  cs_pin: "GPIO15"
+  evu_pin: "GPIO5"
+  entity_room_temperature: "sensor.durchschnittstemperatur_haus_ohne_keller"
```

Comment:
Should we pass this down from the top-level yaml? What do you think? Is more of an overall config, right?

Reply:
ok, I was on the wrong track again ... this is exactly what you are doing ^^